extensions/binding/org.eclipse.smarthome.binding.meteoblue/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: meteoblue Binding
+Bundle-SymbolicName: org.eclipse.smarthome.binding.meteoblue;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.10.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ javax.measure.quantity,
+ com.google.common.collect,
```

Comment:
We are not recommending the use of Guava anymore - please remove this dependency and use Java8 constructs instead.